augly/image/functional.py

```diff
@@ -1588,7 +1588,7 @@ def pixelization(
 
     @returns: the augmented PIL Image
     """
-    assert ratio > 0, "Expected 'ratio' to be a positive number"
+    assert ratio >= 0, "Ratio should be a positive number greater than 0"
```

Comment:
`0` is not a positive number so you can just say "XX should be a positive number". Please revise all these accordingly in this file for consistency. Condition is the same for all, so you can use the same message everywhere. Also why did you allow `0` in asserts?